### PR TITLE
Update ws module version

### PR DESCRIPTION
**Pull request by Fatme**
In order to fix vulnerabilities problems we need to update the version of ws module - https://nodesecurity.io/check/nativescript

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
   ],
   "dependencies": {
     "@types/ora": "1.3.2",
+    "@types/ws": "4.0.1",
```
Comment (rosen-vladimirov) [inline package.json]: this should be devDependency: `npm i --save-dev --save-exact @types/ws`

Comment (rosen-vladimirov) [reply package.json]: btw the @types/ora need to be devDependency as well

**Comment by Fatme** [comment]
run ci

**Comment by Fatme** [comment]
run ci

**Comment by Fatme** [comment]
run ci

**Pull request merged**